### PR TITLE
Update travis files

### PR DESCRIPTION
**Pull request by bwrsandman**

### CODE REVIEW

**Comment by yvaucher** [comment]
~~+1~~
version should be 8.0 as pedro stated it
:-1:

File: .travis.yml [hunk 1]
```diff
@@ -1,47 +1,25 @@
...
-    - "2.7"
-#    - "2.6"  # not supported by odoo 8
+  - "2.7"
 
 env:
-    - ODOO="https://github.com/savoirfairelinux/odoo/archive/setuptools-addons.tar.gz"  # Temp until https://github.com/odoo/odoo/issues/185 or https://github.com/odoo/odoo/issues/441 is fixed
-#    - ODOO="https://github.com/odoo/odoo/archive/master.tar.gz"
-#    - ODOO="https://github.com/OCA/OCB/archive/master.zip"
+  - VERSION="7.0" ODOO_REPO="odoo/odoo"
+  - VERSION="7.0" ODOO_REPO="OCA/OCB"
```
Comment (pedrobaeza) [inline .travis.yml]: Why 7.0 if this is 8.0 branch?

Comment (bwrsandman) [inline .travis.yml]: Good catch, thanks

**Comment by bwrsandman** [comment]
Set to version 8.0

**Comment by pedrobaeza** [comment]
:+1:

**Comment by yvaucher** [comment]
:+1:

**Comment by gurneyalex** [comment]
:+1:

**Pull request merged**